### ISSUE TITLE
Missing `version` in `/eth/v1/validator/blinded_blocks/{slot}`

### ISSUE THREAD

**Issue by moshe-blox** [sigp/lighthouse#3418]
## Description

Blinded block production (`/eth/v1/validator/blinded_blocks/{slot}`) is missing the `version` field in it's response, when it should be "bellatrix" ([spec](https://github.com/ethereum/beacon-APIs/blob/master/apis/validator/blinded_block.yaml#L46-L49)).

## Version

Built from `builder-api` branch.

## Present Behaviour

`/eth/v1/validator/blinded_blocks/{slot}` on Sepolia returns:

```json
{
  "data": <block>
}
```

## Expected Behaviour

Should return:

```json
{
  "version": "bellatrix",
  "data": <block>
}
```

## Steps to resolve

1. Set response `version` to the block's version
2. (Arguably) Consider not omitting version from the response even if it's `phase0` in other routes such as `/eth/v1/validator/blocks/{slot}`

**Comment by macladson** [comment]
To add some more context to this issue:

Currently endpoints which respond with `version` such as:
- `/beacon/blocks/{block}`
- `/debug/beacon/states/{state_id}`
- `/validator/blocks/{slot}`
will only do so when using the `v2` endpoint (this is consistent with the spec for the above endpoints).

> Additionally, when using the `v2` endpoints, `version` should still be getting returned even when it is `phase0`. If there is evidence of this not being the case, please let us know. 

*However*

`eth/v1/validator/blinded_blocks/{slot}` is currently the only `v1` endpoint which returns `version` as part of its response.
Lighthouse uses [an assumption](https://github.com/sigp/lighthouse/blob/stable/beacon_node/http_api/src/version.rs#L17-L20) that all endpoints returning `version` are a `v2` endpoint which does not hold in this particular case, which is the cause of this issue.

**Comment by macladson** [comment]
Fixed in #3429
